src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md

```diff
@@ -32,30 +32,57 @@ The concept of Signed URLs is common across various cloud storage solutions for
 - [Azure Blob](#azure-blob)
 
 
-## SharePoint
+### SharePoint
 
 PDF Services APIs can connect with SharePoint using Signed URIs for downloading and uploading the files from SharePoint site. 
 
+#### Supported URL Prefixes
+
+- `https://<companyname>.sharepoint.com`
```

Comment:
please write, 'company-name', as this is a placeholder.

Reply:
done. Changed to tenant-name